gbpservice/nfp/orchestrator/config_drivers/heat_driver.py

```diff
@@ -1161,6 +1579,65 @@ def apply_config(self, network_function_details):
 
         return stack_id
 
+
+    def apply_heat_config(self, nfp_context):
+        service_details = self.get_service_details_from_nfp_context(nfp_context)
+
+        network_function = nfp_context['network_function']
+        service_profile = service_details['service_profile']
+        service_chain_node = service_details['servicechain_node']
+        service_chain_instance = service_details['servicechain_instance']
+        provider = service_details['provider_ptg']
+        consumer = service_details['consumer_ptg']
+        consumer_port = service_details['consumer_port']
+        provider_port = service_details['provider_port']
+        mgmt_ip = service_details['mgmt_ip']
+
+        auth_token = nfp_context['resource_owner_context']['auth_token']
+        provider_tenant_id = nfp_context['tenant_id']
+        heatclient = self._get_heat_client_v1(provider_tenant_id,
+                                              assign_admin=True)
+        if not heatclient:
+            return None
+        
+        stack_template, stack_params = self._create_node_config_data(
+                    auth_token, provider_tenant_id, 
+                    service_chain_node, service_chain_instance,
+                    provider, provider_port, consumer, consumer_port, 
+                    network_function, mgmt_ip, service_details)
```

Comment:
Since we are writing a new function, we should use dict, instead of passing so many variables.
Prev we did this mistake, its not good to pass so many variables in func.